Added keyboard_remote docker information (#27830)
home-assistant/home-assistant.github.io: source/_integrations/keyboard_remote.markdown

```bash
getfacl /dev/input/event*
```

## Containers

> If you are running Home Assistant Container, you need to pass the input device through to the container. This can be done by running the container in privileged mode, but this is is not ideal for security. Instead you can pass the input device you want to use directly into the container with the `--devices` flag. However, restarting the container or unplugging and replugging your keyboard will break this integration as only the instance of the keyboard that existed when the container first starts will be avaible inside of the container.

Reviewer: ```suggestion
If you are running Home Assistant Container, you need to pass the input device through to the container. This can be done by running the container in privileged mode, but this is not ideal for security. Instead, you can pass the input device you want to use directly into the container with the `--devices` flag. However, restarting the container or unplugging and replugging your keyboard will break this integration. This is because only the instance of the keyboard that existed when the container first started will be available inside the container.
```

> Here is an incompolete example `docker-compose.yml` that allows Home Assistant persistent access to input devices in a container without privileged mode:

Reviewer: ```suggestion
Here is an incomplete example `docker-compose.yml` that allows Home Assistant persistent access to input devices in a container without privileged mode:
```

> Here is an incomplete example `docker-compose.yml` that allows Home Assistant persistent access to input devices in a container without privileged mode:

Reviewer 2: This references "without privileged mode", which is not supported. Please remove that.

```yaml
version: '3.7'

services:
  homeassistant:

>     image: homeassistant/home-assistant:stable

Reviewer 2: We don't recommend the use of DockerHub; or official resource is the GitHub Container Registry.

```suggestion
    image: ghrc.io/homeassistant/home-assistant:stable
```

    volumes:
      - config:/config/

>       - /dev/input:/dev/input/ # this is needed to read input events, even in priv mode

Reviewer: ```suggestion
      - /dev/input:/dev/input/ # this is needed to read input events, even in privileged mode.
```

```bash
getfacl /dev/input/event*
```

## Containers

> If you are running Home Assistant inside of a Docker container, you need to pass the input device through to the container. This can be done by running the container in privileged mode, but this is is not ideal for security. Instead you can pass the input device you want to use directly into the container with the `--devices` flag. However, restarting the container or unplugging and replugging your keyboard will break this integration as only the instance of the keyboard that existed when the container first starts will be avaible inside of the container.

Reviewer 2: Please reference this as the "Home Assistant Container" installation method. Docker is just one of the many available.

To allow replugging to work and for the access to survive container restarts, you can pass the path `"/dev/input/"` to the container in both the `devices` and `volumes` sections. If you get pemission errors, you may also need to add `'c 13:* rmw'` to the device cgroup rules.

> Here is an incompolete example `docker-compose.yml` that allows Home Assistant persistent access to input devices in a container without privileged mode:

Reviewer 2: I'm not a fan of sprinkling our docs with multiple of the same docker-compose snippets, to be honest.

Author: I had this section to be a little more explicit for users who might be running the container using `docker run ...` and not compose. I'm fine removing it if you would like it removed.

```yaml
version: '3.7'

services:
  homeassistant:
    image: homeassistant/home-assistant:stable
    volumes:
      - config:/config/
      - /dev/input:/dev/input/ # this is needed to read input events, even in priv mode
    restart: unless-stopped
    device_cgroup_rules:
      # allow creation of /dev/input/* with mknod, this is not enough on its own and needs mknod to be called in the container
      - 'c 13:* rmw' 
    devices:
      # since input id may change, pass them all in

>       # this is not needed with privileged mode

Reviewer 2: We don't support running unprivileged, so this can be removed.
